src/services/salary/salaryService.ts

```
@@ -0,0 +1,280 @@
+/**
+ * Triggers a salary batch from a schedule.
+ */
+export async function triggerSchedule(scheduleId: string): Promise<void> {
+  const schedule = await prisma.salarySchedule.findUnique({
+    where: { id: scheduleId },
+  });
+
+  if (!schedule || schedule.status !== "active") return;
+
+  const amountConfig = schedule.amountConfig as any[];
+  const totalAmount = amountConfig.reduce((acc, item) => acc.add(new Decimal(item.amount)), new Decimal(0));
```

ESLint

[error] Replace `(acc,·item)·=>·acc.add(new·Decimal(item.amount)),·new·Decimal(0)` with `⏎····(acc,·item)·=>·acc.add(new·Decimal(item.amount)),⏎····new·Decimal(0),⏎··`

(prettier/prettier)